packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -224,11 +246,89 @@ class AuroraClusterEngine extends MySqlClusterEngineBase {
   }
 }
 
+/**
+ * The versions for the Aurora MySQL cluster engine
+ * (those returned by {@link DatabaseClusterEngine.auroraMysql}).
+ */
+export class AuroraMysqlEngineVersion {
```

Comment:
Naming - why is this `AuroraMysql` engine version vs the 1.x `Aurora` engine version?

Reply:
It's just how the engines are named in RDS (`aurora` and `aurora-mysql`), we jus follow their naming.